file_writer_test.go

```diff
@@ -270,6 +265,39 @@ func TestCreateEmptyFile(t *testing.T) {
 	assertPathError(t, err, "create", "/_test/emptyfile", os.ErrExist)
 }
 
+func TestCreateFileExists(t *testing.T) {
+	const filePath = "/_test/create_file_exists"
+
+	client := getClient(t)
+	createFile := func() (*FileWriter, error) {
+		return client.CreateFile(filePath, 1, 1048576, 0755)
+	}
+
+	t.Run("prev file closed", func(t *testing.T) {
```

Comment:
Do you mind writing this to stylistically match the other tests? Sorry for being picky.

Reply:
Reworked, sub-tests were rewritten as separate tests.